_posts/2022-01-20-saying-no.md

```diff
@@ -0,0 +1,43 @@
+
+Having these boundaries clear and upfront is more complex but crucial for unexpected tasks. If you already feel overwhelmed or have too much to do when asked to do something new: you should say "no". If when asked for an estimate, [you can tell there's a bunch of complexity that was not considered by those who requested the estimate]({{ '/2021/11/29/hidden-complexities-of-software-estimation/' | absolute_url }}) and know that your realistic one will disappoint them: you should give them the realistic estimate.
+
+I call this "front-loading disappointment". You're better to have a conversation about your full workload or longer estimate now when people can prepare for it than making them happy now and then disappointing them in a few months when you didn't deliver what they planned for you. I've also found it to be less disruptive for your organisation for you to give an estimated deliverable earlier than planned than later than planned.
+
+### When to say "no"
+
+Ok, we've touched upon why you should say "no", so let's discuss when it's appropriate to do so.
+
+What's your current workload, the stress outside work and the number of commitments made to coworkers, family and friends look like now? What are your goals for the next 3-12 months and what's the best use of your time to achieve them? Regularly asking yourself these questions can help prime you to remind you of them when receiving a request.
+
+When a new request comes in, and you've asked yourself the questions above, you'll probably have a good sense of whether you can meet the needs and expectations of the requester or not. If you think you cannot right now: say "no".
+
+Don't worry about if you've been saying "no" a lot recently; this is not necessarily a problem. Of course, if you're saying no to every request from coworkers, friends and family, this could be a symptom of a broader problem. At work, though, it's pretty expected that there will be more demands on your time than your available time. If you're saying "no" a lot, but you're still delivering great work, and your coworkers and manager are happy with you: you're doing enough.
+
+### How to say "no"
+
+There's still a little bit of art involved in saying "no". It may be that your "no" actually means "no because ..., "no unless you ..." or "no if this ...". These "no"s may be conditional on you as an individual, the timing of the request, the scope of the request or many other reasons. Try to share your reasoning with the person who has requested this of you; it may be that something you understood as critical to the task is actually optional and can be removed given the extra context of a "no because ..." or "no unless ..." provides.
```

Comment:
- 😶 
```suggestion
There's still a little bit of art involved in saying "no". It may be that your "no" actually means "no because ..., "no unless you ..." or "no if this ...". These "no"s may be conditional on you as an individual, the timing of the request, the scope of the request, or many other reasons. Try to share your reasoning with the person who has requested this of you; it may be that something you understood as critical to the task is actually optional and can be removed given the extra context of a "no because ..." or "no unless ..." provides.
```